frontend/app/.lintstagedrc.js

```diff
@@ -2,5 +2,5 @@ const baseConfig = require('../../.lintstagedrc');
 
 module.exports = {
   ...baseConfig,
-  '*.{ts,tsx}': 'yarn stylelint-fix',
+  '*.{js,jsx,ts,tsx}': ['yarn lint-fix', () => 'yarn test-circular'],
```

Comment:
I'm not in favour of adding them in the precommit hook. I think it would be better to add them in the `test` command

Reply:
We removed them from the pre-commit hook (for now ... ;) )